CHANGELOG.md

```diff
@@ -7,60 +7,66 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.7.1] - 2021-05-10
+
+### Added
+
+- Add missing typescript declaration of the PDFWStreamForBuffer
+
 ## [1.7.0] - 2021-03-08
 
 ### Added
 
-* More electron 11.x releases
-* Add electron 12.0.0
-* Add electron 10.2, 10.3
-* Add electron 9.4, 9.3
+- More electron 11.x releases
```

Comment:
Please revert this changes and below. 
if you'd like to lint this file open up a second pr and add a changelog entry.

Thank you for your time.